src/mission.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Stevens Blueprint
```

Comment:
Nit: You can change this title to just be `Mission` to align with the other .md files